x-pack/plugin/snapshot-repo-test-kit/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/10_analyze.yml

```diff
@@ -24,7 +24,7 @@ setup:
         body:
           type: fs
           settings:
-            max_snapshot_bytes_per_sec: "1b"
+            max_snapshot_bytes_per_sec: "1mb"
```

Comment:
I believe you could go even lower here to `10k` (that's what I used to fix the same issue in `org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT#testThrottling` if you wanted to. That would already suffice to never have macroscopic wait times I think. Maybe then we don't even need to change the other parameters and run with more blobs?

Reply:
Hi @original-brownbear , thanks! I changed it to 10k. However, I could not leave the other parameters as they were. Especially the problem was the `max_blob_size: 2gb`, since this made the first blob to take more than >60secs and it times out. So blobs need to be able to finish within a few seconds, so that the reply can come back. Thus, I had to toggle this. I made the max blob size to 500kb, so that with 10kb/s it can finish for surely within 50secs worst-case. Please see latest commit.